cmd/soroban-rpc/internal/db/db.go

```diff
@@ -6,13 +6,16 @@ import (
 	"embed"
 	"errors"
 	"fmt"
-	"strconv"
-	"sync"
-
 	sq "github.com/Masterminds/squirrel"
 	_ "github.com/mattn/go-sqlite3"
 	"github.com/prometheus/client_golang/prometheus"
 	migrate "github.com/rubenv/sql-migrate"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"path"
+	"strconv"
+	"sync"
+	"testing"
```

Comment:
If this is accepted by the linter ... it's buggy

Reply:
hmm yeah I dont see that in [error logs](https://github.com/stellar/soroban-rpc/actions/runs/9606783845/job/26496854173?pr=215)

Reply:
golang-ci is still failing, so I guess there is a problem here. Regardless, stdlib imports should be present in their own block at the very beginning.

Reply:
All lint errors resolved now !! phew